docker/maistra-builder_2.3.Dockerfile

```diff
@@ -76,6 +76,7 @@ RUN go install google.golang.org/protobuf/cmd/protoc-gen-go@${GOLANG_PROTOBUF_VE
     go install github.com/gogo/protobuf/protoc-gen-gogofast@${GOGO_PROTOBUF_VERSION} && \
     go install github.com/gogo/protobuf/protoc-gen-gogofaster@${GOGO_PROTOBUF_VERSION} && \
     go install github.com/gogo/protobuf/protoc-gen-gogoslick@${GOGO_PROTOBUF_VERSION} && \
+    go install github.com/protobuf-tools/protoc-gen-deepcopy@v0.0.3 && \
```

Comment:
I think this just got renamed in upstream image adding the suffix golang in it.

See https://github.com/maistra/test-infra/pull/269/files

If that's the case, changing it in api repo should fix the issue?

Reply:
Thanks, I will try.